TidepoolOnboarding/Views/Sections/WelcomeViews.swift

```diff
@@ -46,12 +46,17 @@ struct WelcomeTabView: View {
                 .accessibilityLabel(String(format: LocalizedString("Tidepool Loop Welcome, page %d of %d", comment: "Onboarding, Welcome section, image accessibility label"), index + 1, welcomeData.count))
                 .alertOnLongPressGesture(enabled: onboardingViewModel.allowDebugFeatures && index == 0,
                                          title: "Are you sure you want to skip the rest of onboarding?") {  // Not localized
-                    onboardingViewModel.skipAllSections(forceSimulators: true)   // NOTE: DEBUG FEATURES - DEBUG AND TEST ONLY
+                    onboardingViewModel.skipAllSections(forceSimulators: false)   // NOTE: DEBUG FEATURES - DEBUG AND TEST ONLY
```

Comment:
With this change you are requiring folks who use this skip feature to change their behavior. If possible, please flip these changes (so long press on the image does exactly what it did before and a new long press on the text does the new behavior without the simulators).